polylogue/pipeline.py

```
@@ -131,13 +149,25 @@ def collect_attachments(
                 if needs_download:
                     ok = drive.download_attachment(att_id, local_path)
                     if not ok:
-                        failures.append(att_id)
+                        try:
+                            rel = str(local_path.relative_to(collector.markdown_root))
+                        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)